src/components/nav-user/nav-user.tsx

```diff
@@ -47,21 +47,17 @@ export class CalciteNavUser {
   render() {
     return (
       <Host>
-        <a tabIndex={0}>
+        <a aria-label={this.label || this.fullName || this.username} tabIndex={0}>
           <calcite-avatar
             full-name={this.fullName ? this.fullName : null}
             thumbnail={this.thumbnail ? this.thumbnail : null}
             user-id={this.userId ? this.userId : null}
           />
 
-          {(this.fullName || this.username) && this.textEnabled && (
```

Comment:
See above - keep the `textEnabled` for now - in the future perhaps this could be part of a "when to hide what" viewport width story, but for now this will allow folks to quickly hide or show the text when needed.

Reply:
makes sense.